{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"reachabilityStartFailed"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/datacapturing\/cyfacesynchronizer\/synchronizererror\/reachabilitystartfailed"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/CyfaceSynchronizer\/SynchronizerError\/reachabilityStartFailed","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"If starting reachability checks fails."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"reachabilityStartFailed"}],"title":"CyfaceSynchronizer.SynchronizerError.reachabilityStartFailed","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:13DataCapturing18CyfaceSynchronizerC0D5ErrorO23reachabilityStartFailedyA2EmF","modules":[{"name":"DataCapturing"}]},"hierarchy":{"paths":[["doc:\/\/DataCapturing\/documentation\/DataCapturing","doc:\/\/DataCapturing\/documentation\/DataCapturing\/CyfaceSynchronizer","doc:\/\/DataCapturing\/documentation\/DataCapturing\/CyfaceSynchronizer\/SynchronizerError"]]},"references":{"doc://DataCapturing/documentation/DataCapturing/CyfaceSynchronizer/SynchronizerError":{"role":"symbol","title":"CyfaceSynchronizer.SynchronizerError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"SynchronizerError"}],"abstract":[{"type":"text","text":"Errors thrown during data synchronization."}],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/CyfaceSynchronizer\/SynchronizerError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"SynchronizerError"}],"url":"\/documentation\/datacapturing\/cyfacesynchronizer\/synchronizererror"},"doc://DataCapturing/documentation/DataCapturing":{"role":"collection","title":"DataCapturing","abstract":[],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing","kind":"symbol","type":"topic","url":"\/documentation\/datacapturing"},"doc://DataCapturing/documentation/DataCapturing/CyfaceSynchronizer":{"role":"symbol","title":"CyfaceSynchronizer","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"CyfaceSynchronizer"}],"abstract":[],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/CyfaceSynchronizer","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"CyfaceSynchronizer"}],"url":"\/documentation\/datacapturing\/cyfacesynchronizer"},"doc://DataCapturing/documentation/DataCapturing/CyfaceSynchronizer/SynchronizerError/reachabilityStartFailed":{"role":"symbol","title":"CyfaceSynchronizer.SynchronizerError.reachabilityStartFailed","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"reachabilityStartFailed"}],"abstract":[{"type":"text","text":"If starting reachability checks fails."}],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/CyfaceSynchronizer\/SynchronizerError\/reachabilityStartFailed","kind":"symbol","type":"topic","url":"\/documentation\/datacapturing\/cyfacesynchronizer\/synchronizererror\/reachabilitystartfailed"}}}